api.xml

```diff
@@ -125,7 +125,46 @@
 	</command>
 		
 
-		<command id='getMenteeMatch' security='protected'>
+	<command id='getMatches' security='protected'>
+		<description><![CDATA[Gets a list of mentor/mentee matches with names and usernames.]]></description>
+
+		<index>
+			<action file='db/db.php'><![CDATA[getMatches();]]></action>
+		</index>
+
+		<get>
+			<action file='db/db.php'><![CDATA[getMatches();]]></action>
```

Comment:
I think you can just keep the index element and it will work. I think the "get" element is supposed to be used to get a specific one (e.g. api/mentor/atorres), whereas the index is supposed to list all of them.